src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -107,7 +107,7 @@ public function has(string $id, string $domain = 'messages')
             return true;
         }
 
-        if (null !== $this->fallbackCatalogue) {
+        if (isset($this->fallbackCatalogue)) {
```

Comment:
no need to change that, as it can indeed be null and no just unset.

Reply:
reverted in 301c7ae454beb38c0de75a0d21fba76a3e69efd3